src/GNDStk/v1.9/key.hpp

```diff
@@ -0,0 +1,119 @@
+
+// THIS FILE WAS AUTO-GENERATED!
+// DO NOT MODIFY!
+
+// This file contains Meta and Child objects for the current GNDS version.
+// If someone wishes to use the GNDS Core Interface with the classes that
+// we've autogenerated in this context, these should prove to be useful.
+
```

Comment:
Does this mean we can remove the definition of the child and meta key names in the core interface and add them to the autogenerated code instead? It makes sense to have these only provided by GNDS standard version.

Reply:
Yes, it means that, and is a good thing. The `Child` and `Meta` objects in the present Core Interface were something I put in, early on, based on what I saw in the existing GNDS files. In other words, they were sort of ad-hoc and probably incomplete, but necessary in order to write functioning codes. Now, with the autogenerator, we can (and should) make them automatically, with one complete set of such objects for each GNDS version.

Right now, the difficulty of removing the original ones is that many (probably most!) of the codes in the test suite depend on these old `Child` and `Meta` objects. So, phasing out the old ones requires planning, to avoid a spectacular breaking of the test system. A good solution might be to put them into, say, a `test` or `testing` namespace that would be excluded from the Core Interface, but kept around indefinitely (probably as an undocumented feature, so as not to confuse users by its existence in the code), for use with the tests.